torch/serialization.py

```diff
@@ -121,8 +121,18 @@ def _with_file_like(f, mode, body):
             f.close()
 
 
+def _is_compressed_file(f):
+    compress_modules = ['gzip']
+    try:
+        return f.__module__ in compress_modules
+    except AttributeError:
+        return False
+
+
 def _is_real_file(f):
```

Comment:
Maybe we should rename this to something else now? Like _should_read_directly(f) or something.

At any rate you should change the comment.